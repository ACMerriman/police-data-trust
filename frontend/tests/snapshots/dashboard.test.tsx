import * as React from "react"
import Dashboard from "../../pages/dashboard"
import renderer from "react-test-renderer"

it("renders Dashboard correctly", () => {
  const tree = renderer.create(<Dashboard />).toJSON()
  expect(tree).toMatchSnapshot()
})
