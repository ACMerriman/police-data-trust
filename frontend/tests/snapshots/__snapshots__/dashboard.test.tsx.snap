// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Dashboard correctly 1`] = `
<div>
  <header
    className="wrapper"
  >
    <div
      className="backgroundBanner"
    >
      <div
        className="leftHeader"
      >
        <div
          className="logoContainer"
        >
          <img
            alt="A blue circular logo design reads 'National Police Data Coalition'"
            className="logo medium"
            src="/NPDCLogo.svg"
          />
        </div>
        <div
          className="titleContainer"
        >
          <h2
            className="mobileTitle"
          >
            N.P.D.C.
          </h2>
          <h2
            className="desktopTitle"
          >
            National Police Data Coalition
          </h2>
          <p>
            The national index of police incidents
          </p>
        </div>
      </div>
      <div
        className="rightHeader"
      >
        <nav
          className="nav"
        >
          <div
            className="mobileNav"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-bars fa-w-14 "
              data-icon="bars"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <div
            className="desktopNav"
          >
            <ul
              className="rightHeader"
              onClick={[Function]}
            >
              <li
                className="tab"
              >
                <a
                  href="#"
                >
                  Search
                </a>
              </li>
              <li
                className=""
              >
                <a
                  href="#"
                >
                  Profile
                </a>
              </li>
              <li
                className=""
              >
                <a
                  href="#"
                >
                  About
                </a>
              </li>
            </ul>
          </div>
        </nav>
        <button
          className="primaryButton"
          type="button"
        >
          DONATE
        </button>
      </div>
    </div>
  </header>
</div>
`;
